examples/subcurrency/src/main.sw

```diff
@@ -69,12 +69,12 @@ impl Token for Contract {
         storage.balances.insert(receiver, storage.balances.get(receiver).unwrap_or(0) + amount);
     }
 
-    #[storage(read, write)]
+    #[storage(read, write), payable]
```

Comment:
Same as my other comment.. how come `payable` is needed here?

Reply:
> Ahh, my bad. I think I understood the use of #payable incorrectly. Thanks for the clarification.

Will make the relevant changes here.